sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -33,6 +33,7 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.test.{SharedSQLContext, SQLTestUtils}
 import org.apache.spark.sql.types._
 
+//noinspection ScalaStyle
```

Comment:
We can disable only the lines with the block as below if you need this for non-ascii characters: 

```scala
// scalastyle:off
...
// scalastyle:on
```